### DISCUSSION TITLE
Is it possible to chose one drone configurations before start the simulation? I mean select which drone to spawn

### DISCUSSION THREAD

**Discussion by Sabrin0** [microsoft/AirSim#4093] [Support Q&A] [answered]
The parameter `'SimMode' : ' ' ` allows to chose only between car or drone, not among more drones with different sensors.

**Comment by jonyMarino** [comment]
@Sabrin0 I don't know if I understand. You want to select wich drone to use before runtime? You can comment/uncomment the drone configurations in the settings.json.

**Reply by Sabrin0** [reply]
Yes,I could act like that but I'm looking for a smart solution. I mean, I'd like to choose directly from Unreal once the simulation starts. Spawn a drone with different sensor equipment depending on the environment

**Reply by jonyMarino** [reply]
You can have a settings.json file for each environment. It has precedence over the one located in the Documents folder.

**Reply by Sabrin0** [reply]
Nice, that sounds great. Thank you

**Comment by jonyMarino** [comment, marked as answer]
You can have a settings.json file for each environment. It has precedence over the one located in the Documents folder.
